_inc/lib/plugins.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Plugins Library
+ *
+ * Helper functions for installing and activating plugins.
+ *
+ * Used by the REST API
+ *
+ * @autounit api plugins
+ */
+
+include_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';
+include_once ABSPATH . 'wp-admin/includes/file.php';
+
+class Jetpack_Plugins {
+
+	 /**
+	 * Install a plugin.
+	 *
+	 * @since 5.8.0
+	 *
+	 * @param string $slug Plugin slug.
+	 *
+	 * @return bool|WP_Error True if installation succeeded, error object otherwise.
+	 */
+	public static function install_plugin( $slug ) {
+		if ( is_multisite() && ! current_user_can( 'manage_network' ) ) {
+			return new WP_Error( 'not_allowed', 'You are not allowed to install plugins on this site.' );
```

Comment:
Is this error message something the user would see? If so, then it should be translatable.